organizations/user.go

```diff
@@ -218,7 +218,7 @@ func UpdateProfilePicture(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	memberDoc, _ := utils.GetMongoDbDoc(member_collection, bson.M{"_id": pMemId, "org_id": orgId})
+	memberDoc, _ := utils.GetMongoDbDoc(member_collection, bson.M{"_id": pMemId, "org_id": pOrgId})
```

Comment:
why do you make this change? did you test it? the pOrgId is a primitive format and the orgId is a string which is the format the org_id is in the doc. This change will have the memberDoc always be nil. Please revert.